client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -351,3 +352,25 @@ function useShowSiteCreationNotice( allSites: SiteExcerptData[], newSiteID: numb
 		window.history.replaceState( null, '', newUrl.toString() );
 	}, [ __, allSites, dispatch, newSiteID ] );
 }
+
+function useShowSiteTransferredNotice() {
+	const { __ } = useI18n();
+	const dispatch = useDispatch();
+	const shownSiteTransferredNotice = useRef( false );
+
+	useEffect( () => {
+		if ( shownSiteTransferredNotice.current ) {
+			return;
+		}
+		const url = new URL( window.location.href );
+		if ( url.searchParams.get( 'transfer_complete' ) === 'true' ) {
+			shownSiteTransferredNotice.current = true;
+
+			dispatch( successNotice( __( 'Your blog transfer succeeded!' ), { duration: 8000 } ) );
```

Comment:
```suggestion
			dispatch( successNotice( __( 'Site transfer complete.' ), { duration: 8000 } ) );
```

@javierarce What do you think about this copy, for consistency with 'Post published' and 'SSH key added.'?

Reply:
Just FYI, again, as in my above comment states, I added it like this, to keep a "consistency" between `my-blogs` and this one.

Reply:
I think in any case, we should use at least "Your site transfer succeeded!" to be consistent with the "site transfer" language that we are using everywhere else

Reply:
I like that @katinthehatsite , I'll go with that!